recipes/uncrustify/all/conanfile.py

```diff
@@ -58,16 +59,15 @@ def package(self):
             os.remove(os.path.join(self.package_folder, "ChangeLog"))
             os.remove(os.path.join(self.package_folder, "HELP"))
             os.remove(os.path.join(self.package_folder, "README.md"))
-            tools.rmdir(os.path.join(self.package_folder, "cfg"))
-            tools.rmdir(os.path.join(self.package_folder, "doc"))
-
-        tools.rmdir(os.path.join(self.package_folder, "share"))
-
-    def package_id(self):
-        del self.info.settings.compiler
+            rmdir(self, os.path.join(self.package_folder, "cfg"))
+            rmdir(self, os.path.join(self.package_folder, "doc"))
 
     def package_info(self):
+        self.cpp_info.includedirs = []
+        self.cpp_info.libdirs = []
+        self.cpp_info.resdirs = []
+
+        # TODO: to remove in conan v2
         binpath = os.path.join(self.package_folder, "bin")
         self.output.info(f"Adding to PATH: {binpath}")
         self.env_info.PATH.append(binpath)
```

Comment:
Isn't it `runenv_info` here ? https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#removed-self-env-info

Reply:
`PATH` is a first citizen in conan v2, `cpp_info.bindirs` are automatically injected to `PATH`.

Reply:
Documented here: https://docs.conan.io/en/latest/reference/conanfile/tools/env/virtualrunenv.html